plugins/PrivacyManager/IPAnonymizer.php

```diff
@@ -25,6 +26,11 @@ class IPAnonymizer
      */
     public static function applyIPMask(IP $ip, $maskLength)
     {
+        // IPv6 can't handle anonymizing 4 bytes
+        if ($maskLength == 4 && $ip instanceof IPv6 && !$ip->isMappedIPv4()) {
```

Comment:
@sgiehl wouldn't we add this maybe to the IPv6 class? (I suppose IPv4 already supports it)

Reply:
created https://github.com/matomo-org/component-network/pull/15
We will need to publish a new release and update the component here after merging...

Reply:
👍  left a comment in that PR. Feel free to merge the component PR afterwards, create the release and then update this PR

Reply:
updated